src/commands.ts

```diff
@@ -239,6 +242,9 @@ export async function deployClasses() {
   await generateSdks(functionUrlForFilePath);
 
   reportSuccess(classesInfo, configuration);
+  
+  let projectId = classesInfo[0].projectId
```

Comment:
Add this in the `reportSucess` method.

Reply:
it can't be added, the method its also used for local env